### DISCUSSION TITLE
Download Url

### DISCUSSION THREAD

**Discussion by g4gooku** [rayanfer32/TgindexPro#4] [Q&A] [answered]
Does the download url always work or expire in some time?

**Comment by rayanfer32** [comment, marked as answer]
I made this repo to make the download links permanent
The DL and VLC buttons provide permanent links

**Reply by g4gooku** [reply]
That's great!
But when i deploy this project am facing error do you have any video guide bro?

**Comment by rayanfer32** [comment]
Tell me what error are you getting. Ill help you out

**Reply by g4gooku** [reply]
Your telegram username bro?

**Reply by rayanfer32** [reply]
[t.me/rayanfer](http://t.me/rayanfer)

**Comment by g4gooku** [comment]
Facing this error bro in SESSION_STRING
![Capture](https://user-images.githubusercontent.com/76486699/103098676-a6c1e980-4631-11eb-9d05-87cbb0f66942.PNG)

**Reply by rayanfer32** [reply]
You cannot keep session string blank. .. It was cause .env read error
